test/query.test.js

```diff
@@ -1568,13 +1569,13 @@ describe('query', () => {
     return Promise.all([p1, p2, p3, p4])
   })
 
-  test('contains', () => {
-    var obj = { a: { b: 1 } }
-    var p1 = assertQuery(query.Contains(['a', 'b'], obj), true)
-    var p2 = assertQuery(query.Contains('a', obj), true)
-    var p3 = assertQuery(query.Contains(['a', 'c'], obj), false)
-    return Promise.all([p1, p2, p3])
-  })
+  // test('contains', () => {
```

Comment:
@marrony This no longer passes either. I'm assuming it's because we're removing it in the next API version?

Reply:
you can just remove this test, `Contains` function will be removed on api v4